WordPress/gutenberg
# Add options for debugging php unit tests in package.json

## What?
Adds `:debug` options to the php unit test scripts.

## Why?
Allows easy debugging of php unit tests via the wp-env xdebug interface.

## How?
Adds a `wp-env` startup option that passes in the `--xdebug` flag and a `test:unit:php:debug` and `test:unit:php:multisite:debug` that makes use of it

## Testing Instructions
 - Set up an xdebug listener, eg. with the following debug launch settings in VS Code:

```
	{
		"name": "Listen for XDebug",
		"type": "php",
		"request": "launch",
		"port": 9003,
		"pathMappings": {
			"/var/www/html/wp-content/plugins/gutenberg": "${workspaceFolder}/"
		}
	}
```
 - Put some breakpoints in one of the php unit tests
 - Run `npm run test:unit:php:debug` and check that the tests pause on the breakpoints
 - Check that `test:unit:php` and `test:unit:php:multisite` still complete as expected
